frontends/web/src/routes/buy/pocket-terms.tsx

```diff
@@ -46,22 +46,22 @@ export const PocketTerms = ({ onAgreedTerms }: TProps) => {
         <h2 className={style.title}>{t('buy.pocket.security.title')}</h2>
         <p>{t('buy.pocket.security.p1')}</p>
         <p>
-          <A className={style.link} href="https://bitbox.swiss/bitbox02/threat-model/">
+          <A href="https://bitbox.swiss/bitbox02/threat-model/">
             {t('buy.pocket.security.link')}
           </A>
         </p>
         <h2 className={style.title}>{t('buy.pocket.kyc.title')}</h2>
         <p>{t('buy.pocket.kyc.p1')}</p>
         <p>
-          <A className={style.link} href="https://pocketbitcoin.com/faq">
+          <A href="https://pocketbitcoin.com/faq">
             {t('buy.pocket.kyc.link')}
           </A>
         </p>
 
         <h2 className={style.title}>{t('buy.pocket.data.title')}</h2>
         <p>{t('buy.pocket.data.p1')}</p>
         <p>
-          <A className={style.link} href="https://pocketbitcoin.com/policy/privacy">
+          <A href="https://pocketbitcoin.com/policy/privacy">
             {t('buy.pocket.data.link')}
           </A>
         </p>
```

Comment:
![Screen Shot 2023-07-27 at 09 49 37](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/f9f0e369-eaa6-4875-858f-718f284783ec)
![Screen Shot 2023-07-27 at 09 49 15](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/b072c8ca-c5e2-4c83-8b4d-cae81c1e34f8)

Visual change: in the info popup, the links were not blue, now they are.

Reply:
![Screen Shot 2023-07-27 at 09 49 44](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/049864ee-32e8-4d39-8484-e26de7fe7909)

![Screen Shot 2023-07-27 at 09 49 26](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/02d08632-1282-400a-9af7-febf1c3a4f66)

(these links were already blue in 4.38.0)

Reply:
More of a question, how is the decision on the link colour made? I assume all should be blue + underlined, but some (maybe when used on a blue background, or if it's a "delete" link)  it'll be custom styled..?